packages/1155-contracts/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@zoralabs/zora-1155-contracts",
-  "version": "2.1.0",
+  "version": "2.1.1-premint-api.0",
   "repository": "git@github.com:ourzora/zora-protocol.git",
   "author": "Iain <iain@zora.co>",
   "license": "MIT",
-  "main": "./dist/index.cjs",
+  "main": "./dist/index.js",
```

Comment:
should we remove this `main` and `types` since this now just  exports contracts?

Reply:
This also exports the wagmi types?